lib/msal-browser/src/utils/BrowserConstants.ts

```diff
@@ -186,6 +186,7 @@ export const KEY_FORMAT_JWK = "jwk";
 export const WrapperSKU = {
     React: "@azure/msal-react",
     Angular: "@azure/msal-angular",
+    FirstParty: "@azure/msal-javascript-1p"
```

Comment:
How will this work when using both 1P and React or Angular?

Reply:
I would try to avoid changing the eSTS SKU as they build some logic around the value AFAIK. My suggestion is to replicate the behavior of the React/Angular SKUs for 1P (rather than making it a distinct option on the existing logic) so that if both are used they are both present on the server telemetry fields.